Terminal.Gui/Input/Key.cs

```diff
@@ -208,13 +209,13 @@ public KeyCode KeyCode
         get => _keyCode;
         init
         {
-#if DEBUG
-            if (GetIsKeyCodeAtoZ (value) && (value & KeyCode.Space) != 0)
-            {
-                throw new ArgumentException ($"Invalid KeyCode: {value} is invalid.", nameof (value));
-            }
+//#if DEBUG
```

Comment:
I strongly disagree with removing this debug check.

I put it in here because it forces drivers to be correct. Please explain the situaiton where you think it's ok for a driver to provide an invalid key code.

Reply:
Because the `TextField` and the `TextView` wasn't processing accented characters because this method always returning false. If you type in the `WindowsDriver` for e.g. the letter '`á`' the virtual key (`ConsoleKey`) is '`A`'. Thus I think that debug check doesn't make sense, sorry.

Reply:
I need to you to be much more precise. 

Instead of making this about TextField/View, what about the `Keys` scenario?

I had 'á' working great with the simple fix I suggested yesterday and it did not require disabling this very sane debug check.

Reply:
Reverted and working, thanks.